src/pages/iou/IOUModal.js

```diff
@@ -159,18 +163,27 @@ class IOUModal extends Component {
     getTitleForStep() {
         const currentStepIndex = this.state.currentStepIndex;
         if (currentStepIndex === 1 || currentStepIndex === 2) {
+            const formattedAmount = this.props.numberFormat(
+                this.state.amount, {
+                    style: 'currency',
+                    currency: this.state.selectedCurrency.currencyCode,
+                },
+            );
+            if (this.props.iouType === 'send') {
```

Comment:
NAB: Should we have the IOU type `send` in a constant?

Reply:
Good point! I have another PR dependent on this one, so will make that change there.